tests/e2e/driver_tests.py

```diff
@@ -630,6 +632,115 @@ def test_initial_namespace(self):
             cursor.execute("select current_database()")
             self.assertEqual(cursor.fetchone()[0], table_name)
 
+class PySQLStagingIngestionTestSuite(PySQLTestCase):
```

Comment:
Can you add more tests?
- Two Puts to the same staging path should fail the second one
- Put/Get/Delete files in another user's staging location should fail with permission-denied errors
- Empty `localPath` argument in the query should fail with parsing errors on server side
- Invalid `stagingPath` argument in the query should fail with parsing errors on server side
  - Valid staging path should be a valid URI starting with `stage://tmp/`

cc @HSCandYH

Reply:
+1

Reply:
Done!

> Two Puts to the same staging path should fail the second one: c8a64c78d8d9ac0477e2dbf582d03d16239cb7b5
> Put/Get/Delete files in another user's staging location should fail with permission-denied errors: d48d3f367392fd4cf1f166210c45c609a28d3002
> Empty localPath argument in the query should fail with parsing errors on server side: e0037e0a8a5e5b224f3d031ec6de6a5f330470e8
> Invalid stagingPath argument in the query should fail with parsing errors on server side: 3fa5d84b7e3f4b25e3ce8826e7e2bc0b66e2bdee